package.json

```diff
@@ -1,146 +1,145 @@
-        },
-        {
-            "name": "Tim Kim",
-            "email": "timk@adobe.com"
-        },
-        {
-            "name": "Benn Mapes",
-            "email": "Benn.Mapes@gmail.com"
-        },
-        {
-            "name": "Michael Wolf",
-            "email": "Michael.Wolf@Cynergy.com"
-        },
-        {
-            "name": "Andrew Grieve",
-            "email": "agrieve@chromium.org"
-        },
-        {
-            "name": "Bryan Higgins",
-            "email": "bhiggins@blackberry.com"
-        },
-        {
-            "name": "Don Coleman",
-            "email": "dcoleman@chariotsolutions.com"
-        },
-        {
-            "name": "Germano Gabbianelli",
-            "email": "tyron.mx@gmail.com"
-        },
-        {
-            "name": "Ian Clelland",
-            "email": "iclelland@chromium.org"
-        },
-        {
-            "name": "Lucas Holmqust",
-            "email": "lholmqui@redhat.com"
-        },
-        {
-            "name": "Matt LeGrand",
-            "email": "mlegrand@gmail.com"
-        },
-        {
-            "name": "Michal Mocny",
-            "email": "mmocny@gmail.com"
-        },
-        {
-            "name": "Sam Breed",
-            "email": "sam@quickleft.com"
-        },
-        {
-            "name": "Tommy-Carlos Williams",
-            "email": "tommy@devgeeks.org"
-        },
-        {
-            "name": "Rubén Norte",
-            "email": "rubennorte@gmail.com"
-        },
-        {
-            "name": "Germano Gabbianelli",
-            "email": "tyrion.mx@gmail.com"
-        },
-        {
-            "name": "Steven Gill",
-            "email": "stevengill97@gmail.com"
-        },
-        {
-            "name": "Jesse",
-            "email": "purplecabbage@gmail.com"
-        }
-    ],
-    "license": "Apache-2.0"
-}
\ No newline at end of file
+  "name": "cordova",
+  "version": "6.2.0-dev",
+  "preferGlobal": "true",
+  "description": "Cordova command line interface tool",
+  "main": "cordova",
+  "engines": {
+    "node": ">=0.9.9"
+  },
+  "engineStrict": true,
+  "bin": {
+    "cordova": "./bin/cordova"
+  },
+  "scripts": {
+    "test": "node node_modules/jasmine-node/bin/jasmine-node --captureExceptions --color spec",
+    "cover": "node node_modules/istanbul/lib/cli.js cover --root src --print detail node_modules/jasmine-node/bin/jasmine-node -- spec"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://git-wip-us.apache.org/repos/asf/cordova-cli.git"
+  },
+  "bugs": {
+    "url": "https://issues.apache.org/jira/browse/CB",
+    "email": "dev@cordova.apache.org"
+  },
+  "keywords": [
+    "cordova",
+    "client",
+    "cli"
+  ],
+  "dependencies": {
+    "cordova-lib": "6.1.0",
+    "cordova-common": "^1.1.0",
+    "q": "1.0.1",
+    "nopt": "3.0.1",
+    "underscore": "1.7.0",
+    "update-notifier": "^0.5.0"
+  },
+  "devDependencies": {
+    "istanbul": "^0.3.4",
+    "grunt": "^1.0.1",
```

Comment:
doing a major jump for grunt removes the warnings eh? Why are we using grunt? looks like someone wanted to use grunt-retire but then the gruntFile & grunt task never get loaded. I suggest we just delete grunt from this repo.